### ISSUE TITLE
Yolo v8

### ISSUE THREAD

**Issue by jafioti** [jafioti/luminal#16]

**Comment by AdrianEddy** [comment]
YOLOv10 https://github.com/THU-MIG/yolov10

**Comment by jafioti** [comment]
Just finished whisper which involved some work on the conv layers. Should be pretty straightforward to do yolo now

**Comment by xnorpx** [comment]
Btw there is a ongoing Yolo9MIT rewrite here: https://github.com/WongKinYiu/YOLO

**Comment by jafioti** [comment]
Yeah sorry been backed up, the yolo example isn't complete yet. should be done very soon

**Comment by jafioti** [comment]
yolo v11 merged
